frontend/src/models/Inspection.ts

```diff
@@ -9,7 +9,7 @@ export interface Inspection {
     inspectionTarget: Position
     videoDuration?: number
     analysisType?: string
-    inspectionUrl?: string
+    inspectionUrl?: string | null
```

Comment:
I'm probably misunderstanding something about typescript, but isn't this implied with the question mark? Or does the question mark indicate "| undefined"?

Reply:
This was me trying to debug something, good catch thanks!